src/main/java/com/google/devtools/build/lib/rules/android/AndroidBinary.java

```diff
@@ -60,10 +60,7 @@
 import com.google.devtools.build.lib.collect.nestedset.NestedSetBuilder;
 import com.google.devtools.build.lib.collect.nestedset.Order;
 import com.google.devtools.build.lib.concurrent.ThreadSafety.Immutable;
-import com.google.devtools.build.lib.packages.BuildType;
-import com.google.devtools.build.lib.packages.RuleClass;
-import com.google.devtools.build.lib.packages.TriState;
-import com.google.devtools.build.lib.packages.Type;
+import com.google.devtools.build.lib.packages.*;
```

Comment:
fyi we can't use wildcard imports, i can fix this on import I think

Reply:
Sorry had my IDE setting misconfigured